### PR TITLE
Drop stack Windows CI

### PR DESCRIPTION
**Pull request by pepeiborra**
The Stack Windows build is problematic:
  https://github.com/haskell/ghcide/pull/922

Stack is already covered by the Azure CI
Windows is already covered by the Github Actions CI

### CODE REVIEW

File: .azure/windows-stack.yml [hunk 1]
```diff
@@ -1,41 +0,0 @@
-jobs:
```
Comment (jneira) [inline .azure/windows-stack.yml:1]: Agree in drop it temporary, but i would keep the config file and only remove the build from azure-pipelines.yaml

File: stack-windows.yaml [hunk 2]
```diff
@@ -1,32 +0,0 @@
-resolver: nightly-2020-06-19
```
Comment (jneira) [inline stack-windows.yaml:1]: Same with this file, i would like to make it work.

File: .azure/windows-stack.yml (hunk 1 above)
Comment (pepeiborra) [reply .azure/windows-stack.yml:1]: You can revert this commit to bring it back

File: stack-windows.yaml (hunk 2 above)
Comment (pepeiborra) [reply stack-windows.yaml:1]: Use git revert

**Review by jneira — APPROVED** [review]

**Comment by pepeiborra** [comment]
Damn it, the rebase Brough `stack-windows.yaml` back. I will need to send another PR to delete it

**Comment by jneira** [comment]
die hard 😝

**Pull request merged**